sellstrom.php

```diff
@@ -806,12 +846,24 @@ public function hookBackofficeHeader($params)
 		$this->processForm($params);
 
 		$content_data = array();
-		$content_data['ps_version'] = _PS_VERSION_;
+		$content_data['ps_version'] = (float)_PS_VERSION_;
```

Comment:
This line is strange by the behaviour you could have in result:
Why don't you replace it with the following one:
```php
$content_data['show_panel'] = version_compare($content_data['ps_version'], '1.5', '>=');
```